cves/CVE-2013-4444.yml

```diff
@@ -43,15 +43,23 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+  This vulnerability allows attackers to upload and execute JavaScript programs
+  on a Tomcat server. The Tomcat security team views this vulnerability as
+  important rather than critical because the circumstances required for it to
+  occur are limited.
+
+  The implementation allowed unrestricted reading of files without obvious
```

Comment:
Was it not obvious that files were checked for scripting? Or were there checks missing to catch even obvious scripting? This sentence is a little confusing.

Reply:
reworded. Thanks!